3-JSS/styles.js

```diff
@@ -0,0 +1,76 @@
+import jss from "jss";
+import preset from "jss-preset-default";
+
+console.log('what?');
+console.log({ jss })
+
+jss.setup(preset());
+const styles = {
+    body: {
+        margin: 0,
+        padding: 0,
+    },
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  taskListContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    textAlign: 'center',
+    border: "2px solid",
+    borderRadius: "40px",
```

Comment:
prefer percentages over big pixel numbers, but if that's the design, that's the design :D